docs/src/docs/hooks/use-media-query.mdx

```diff
@@ -19,7 +19,9 @@ use-media-query hook allows to subscribe to media queries.
 It receives media query as an argument and returns true
 if given media query matches current state.
 Hook relies on `window.matchMedia()` [API](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia)
-and will always return false if api is not available (e.g. during server side rendering).
+and will return false if api is not available.
+When server side rendering, it is important to pass ssrInitialValue because without it the server's initial state will fallback to false, but the client will initialize to the result of the media query. 
```

Comment:
It says to pass ssrInitialValue value, but does not show how to do that with code. Server side rendering must be described in a separate section.

Reply:
Added ssr section to docs